runtime/src/test/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure we need this if we're Spring Data JPA as in https://spring.io/guides/gs/accessing-data-jpa/.